### PR TITLE
Twilio whatsup support

### PR DESCRIPTION
**Pull request by guidone**

### CODE REVIEW

File: lib/platforms/twilio/twilio.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ var Twilio = new ChatExpress({
   routes: {
     '/redbot/twilio': function(req, res) {
       this.receive(req.body);
-      res.send(200);
+      res.send(''); // sending 200 cause an "OK" in whatsup
```
Comment (jekkos) [inline lib/platforms/twilio/twilio.js]: I treid to apply this fix locally but that stopped the service from working completely, so not sure if that will fix it

Comment (guidone) [reply lib/platforms/twilio/twilio.js]: @jekkos sorry for the delay, I've tried and it works, how did you configured your twilio account?

Comment (jekkos) [reply lib/platforms/twilio/twilio.js]: I'm using whatsapp through the sandbox currenttly, as this API is not yet open completely.

Comment (jekkos) [reply lib/platforms/twilio/twilio.js]: So I'm just using the twilio sender/receiver with the sandbox as a sender number.

**Pull request merged**